debug/src/dim2.rs

```diff
@@ -37,7 +43,11 @@ fn create_debug_sprites(
                         .spawn_bundle(create_shape(
                             body,
                             collider.shape(),
-                            (*debug_color).into(),
+                            if sensor_option.is_some() {
+                                debug_color.sensor
+                            } else {
+                                debug_color.physics
+                            },
```

Comment:
If there is no `SensorShape` it can still be a sensor if the rigid body type is `RigidBody::Sensor`.

Reply:
Maybe it'd be cool to add a couple of tests for this.

Reply:
Do you think that adding a system that inserts a `SensorShape` to the entity when a `RigidBody::Sensor` is a bad workaround? Or maybe is better to just check the body type when creating the debug sprite?